### DISCUSSION TITLE
Oqtane Helpdesk Module Issues

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2377] [Q&A]
Hi,

I've installed [Syncfusion HelpDesk Module](https://github.com/ADefWebserver/SyncfusionHelpDesk) in Oqtane and following issues are noticed.

Listed Syncfusion controls and even other Syncfusion controls are not working! 
1. DropDownList
2. DatePicker
3. Edit & Delete Grid Commands are not triggering the event

Kindly guide me to rectify these problems. Please find the attached screenshots for your reference.

**Used Versions:**
**Oqtane:** 3.1.4 
**Syncfuision:** 20.2.0.43

Thanks & Regards!

![helpdesk-module-issue-1](https://user-images.githubusercontent.com/42769344/185734557-1efc0a45-967d-424f-a589-bf9d3292132b.PNG)

![helpdesk-module-issue](https://user-images.githubusercontent.com/42769344/185734434-e284ae0e-3c9e-47cf-827f-97aa25e531b8.PNG)

**Comment by leigh-pointer** [comment]
Read this blog should help https://blazorhelpwebsite.com/ViewBlogPost/47

**Reply by Sharafudeen** [reply]
Previous version of Oqtane and Syncfusion was working fine which will have 0.js & 1.js javascript files. 
Recently I've configured Syncfusion (20.2.0.43) and noticed in the scripts folder and **_it doesn't contains these files (0.js & 1.js) in the package_**. This might be the problem? Even tried to delete those files but no luck!

**Comment by Sharafudeen** [comment]
I've resolved the issues by adding **Syncfusion JS files in the HostResources.cs**

But no luck while trying to add dynamic JS files (find below screenshot).

![helpdesk-module-issue2](https://user-images.githubusercontent.com/42769344/185749600-bbf31007-ad29-4b1b-bfc6-47dccc0134f7.PNG)

**Reply by oqtanecms** [reply]
I only added syncfusion-blazor.min.js in resources,it worked good
![resource](https://user-images.githubusercontent.com/17893776/185782226-1f2f69df-f374-470a-99ca-54ad25463191.PNG)
![resource1](https://user-images.githubusercontent.com/17893776/185782229-b2bf9cc8-1727-4f2d-8cdd-2c0f55833350.PNG)
![resource1](https://user-images.githubusercontent.com/17893776/185782231-1c95ac42-fc88-4e71-addf-3bd694ddbb5d.PNG)
![resource](https://user-images.githubusercontent.com/17893776/185782232-032b85c5-3da5-45de-af32-cbef4305069f.PNG)
.

**Reply by Sharafudeen** [reply]
If you are not including other JS files and some use cases may arise and might face an issues in the future.. but not sure..!

**Comment by leigh-pointer** [comment]
Yiu can do exactly the same thing in a thema or module with out touching the core framework codebase

**Comment by Sharafudeen** [comment]
I've configured Oqtane Default theme in my page and added Oqtane Syncfusion module but some controls overlapping.. I'm wondering Syncfusion also built on Bootstrap 5 then why should we include Syncfusion JS files?
